object/user.go

```diff
@@ -73,7 +73,7 @@ type User struct {
 	LastSigninTime string `xorm:"varchar(100)" json:"lastSigninTime"`
 	LastSigninIp   string `xorm:"varchar(100)" json:"lastSigninIp"`
 
-	Github        string `xorm:"varchar(100)" json:"github"`
+	GitHub        string `xorm:"github varchar(100)" json:"github"`
```

Comment:
Why use GitHub here is that GitHub should be used when getting fields value for reflection. And I added field annotations to xorm, it will not destroy the original database format.